pkg/cli/global.go

```diff
@@ -20,11 +20,13 @@ import (
 
 const (
 	AcceptedLicenseVersions = "license.acceptedVersions"
+	IncognitoMode           = "incognito"
```

Comment:
```suggestion
	IncognitoMode           = "incognito.mode"
```

Just in case there would be a "key" or something like that, so a new field related to incognito settings, then it could more easily be added with `incognito.key` instead of having to introduce a new field like `incognitoKey`.

Reply:
But then it should rather be `incognito.enabled`, shouldn't it? I like my bikeshed yellow, so... :)